ui/shared/AccountSelector.qml

```diff
@@ -149,10 +149,9 @@ Item {
                     id: accountName
                     text: name
                     elide: Text.ElideRight
-                    anchors.right: parent.right
-                    anchors.left: parent.left
                     font.pixelSize: 15
                     height: 22
+                    width: dropdownWidth - (Style.current.padding + iconImg.width + 14 + txtFiatBalance.width + 4 + fiatCurrencySymbol.width + Style.current.padding)
```

Comment:
I think it's probably easier to just modify the anchors so that you set where the text begins and where it elides using its neighbors instead of doing those calculations that will break if we change on component's margin.
Same for the other calculation above

Reply:
Fixed, thank you!